torch-neuronx/training/tp_dp_gpt_neox_20b_hf_pretrain/tp_dp_gpt_neox_20b_hf_pretrain.py

```diff
@@ -0,0 +1,659 @@
+# coding=utf-8
```

Comment:
Is it possible to re-use the existing training script from GPT-NeoX-6.9B, you can then parameterize it with some config json. If 6.9B config is passed, then it would pick 6.9B mmodle and if 20B is passed then it would pick 20B model. That way 6.9B model can also enjoy the SP+TP changes that you added.

Reply:
Have cleaned up the code by deleting unnecessary codes.